### ISSUE TITLE
Wiremill configuration and GCYM

### ISSUE THREAD

**Issue by MorphRed** [Nomi-CEu/nomi-ceu#318]
I had a small accident with my wires today (lol) but I realized that GCYM's large wiremill factory does not have a distinct bus option, so it means it has an incompatibility with the new programmed circuit recipe of wires.
I'm not sure if it's actually the right place to mention as it's more of a gcym issue but i thought it would be good to mention.
Basically, we need distinct buses now
![Screenshot (316)](https://user-images.githubusercontent.com/111656218/230765788-88b07c66-04b4-43bf-b016-29bd36eef302.png)

**Comment by Folanlron** [comment]
Already done, just waiting on CEu and GCYM too release updates..

https://github.com/GregTechCEu/gregicality-multiblocks/pull/44
